app/components/candidate_interface/other_qualifications_review_component.rb

```diff
@@ -115,7 +115,7 @@ def grade_row(qualification)
     end
 
     def grade_set_key(qualification)
-      if qualification.qualification_type == 'non_uk'
+      if qualification.qualification_type == 'non_uk' || (qualification.qualification_type == 'Other' && qualification.qualification_type_name != 'BTEC')
```

Comment:
Should the corresponding spec be updated in this instance?

Reply:
Yep, I've just added a test for this. Cheers Toby